src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,120 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_SIZE = 16;
+    private static final int RESIZE_CONSTANT = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int threshold;
```

Comment:
it's a useless field which value is gonna be read only once, can easily be calculated